.gitignore

```diff
@@ -7,3 +7,5 @@ tcpServer
 
 # vim cache files
 *.swp
+
+config
```

Comment:
I actually meant the database directory, but this is also fine. The user will have to create her own config.